test/dialects/snowflake_test.py

```diff
@@ -11,86 +11,10 @@
 @pytest.mark.parametrize(
     "segment_cls,raw",
     [
-        ("UseStatementSegment", 'USE ROLE "MY_ROLE";'),
```

Comment:
Why remove these test cases?

Reply:
I moved them to a file based test rather than being here. As per the note a few lines above, the `.sql`/`.yml` tests are more robust than these ones, unless they're testing parts of segments rather than a full file.